From 00f8ad41c78df71c464e98d2c1171a40f205db78 Mon Sep 17 00:00:00 2001
From: Justin Ruggles <justin.ruggles@gmail.com>
Date: Mon, 5 Nov 2012 15:36:19 -0500
Subject: [PATCH 194/278] add 24-bit FLAC encoding to Changelog

Also move a misplaced line from the 9_beta2 section to <next>
---
 Changelog |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/Changelog b/Changelog
index e2c4273..f8079d1 100644
--- a/Changelog
+++ b/Changelog
@@ -2,9 +2,13 @@ Entries are sorted chronologically from oldest to youngest within each release,
 releases are sorted from youngest to oldest.
 
 version <next>:
+- ashowinfo audio filter
+- 24-bit FLAC encoding
+
+
+version 9_beta2:
 - metadata (INFO tag) support in WAV muxer
 - support for building DLLs using MSVC
-- ashowinfo audio filter
 
 
 version 9_beta1:
-- 
1.7.9.5

